File: 2029.py
while True:
    try:
        v = float(input())
        d = float(input())
    except EOFError:
        break

    r = d / 2
    h = v / (3.14 * r * r)
    print(f"ALTURA = {h:.2f}")

    area = 3.14 * r * r
    print(f"AREA = {area:.2f}")

File: 2029.java
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		while(sc.hasNextDouble()) {
			double v = sc.nextDouble();
			double d = sc.nextDouble();
			
			double r = d / 2.0;
            double area = 3.14 * r * r;
            double h = v / area;

            System.out.printf("ALTURA = %.2f\n", h);
            System.out.printf("AREA = %.2f\n", area);
		}
		sc.close();
	}
}